browser/src/control/Control.Command.js

```diff
@@ -82,31 +89,13 @@ L.Map.include({
 			if (highlight)
 				message.push('<li>' + highlight + '</li>');
 		});
-		message.push('</ul>', '</div>', '<div>');
+		message.push('</ul>', '</div>', '</div>');
 
 		message = message.join('');
-
-		this.uiManager.showInfoModal('unlock-features-popup', null, ' ', ' ', _('Unlock'), function() {
-			window.open(this.Locking.unlockLink, '_blank');
-			this.uiManager.closeModal(this.uiManager.generateModalId('unlock-features-popup'));
-		}.bind(this), true);
-
-		var paraTag = document.getElementById('unlock-features-popup').querySelectorAll('p')[0];
```

Comment:
it can be easier fixed -> using LayoutingTask
The problem here is: showInfoModal is non-blocking function now.
It will create dialog later in the animation frame.

To do something after dialog is present, schedule next task:
`app.layoutingService.appendLayoutingTask(() => { here old code });`